### PR TITLE
Assignment 1 complete

### PR DESCRIPTION
**Pull request by dwalkes**

### CODE REVIEW

File: student-test/assignment1/Test_validate_username.c [hunk 1]
```diff
@@ -14,9 +14,9 @@
...
-    /**
-     * TODO: Replace the line below with your code here as described above to verify your /conf/username.txt 
-     * config file and my_username() functions are setup properly
-     */
-    TEST_ASSERT_TRUE_MESSAGE(false,"AESD students, please fix me!");
+    char* username = malloc_username_from_conf_file(); 
+
+    TEST_ASSERT_EQUAL_STRING_MESSAGE(my_username(), username, "PASS!");
+
+    free(username);
```
Comment (dwalkes) [inline student-test/assignment1/Test_validate_username.c:21]: @neohere97 thanks for remembering the free() here!

File: test.sh [hunk 2]
```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```
Comment (dwalkes) [inline test.sh:1]: @neohere97 why did you add this in https://github.com/cu-ecen-aeld/assignment-1-neohere97/commit/c80f7657f88c01bc57987a568ccf06ae393e6cae and where did you find it?  This is an old script from previous semesters which shouldn't be used anymore.

**Review by dwalkes — COMMENTED** [review]
Nice work! See comments here...

File: test.sh (hunk 2 above)
Comment (neohere97) [reply test.sh:1]: @dwalkes I found it in the assignments-base code which we had cloned. I followed the instructions on one of the github repository/coursera to add the test-basedir.sh to the base repository as test.sh.

Comment (dwalkes) [reply test.sh:1]: You mean you followed the instructions in https://github.com/cu-ecen-aeld/assignment-autotest ?  This one is already setup for you as a submodule at https://github.com/cu-ecen-aeld/aesd-assignments so there's nothing you need to do for that one.  The instructions in Coursera don't reference assignment-autotest, just aesd-assignments
